uiowa/uiowa
# Remove LBS checkbox, again.

Resolves #2357

In the absence of a fieldset, which I don't think is possible, I added some text to delineate the styles section of the off-canvas form. I called it "Appearance" but open to suggestions or removal. I also added some classes in case we want to add admin styles to differentiate these somehow.

### Testing
- Check out this branch locally.
- Add various blocks to a page via LB.
- See that the styles render below a div with text "Appearance".
  - [ ] Accept, edit or remove text.
  - [ ] Add our own admin form styles or remove applied classes.
- Ensure adding and removing styles is intuitive, works correctly and that https://github.com/uiowa/uiowa/issues/2357 is no longer an issue.
